myriad-executor/src/main/java/org/apache/myriad/executor/MyriadExecutorAuxService.java

```diff
@@ -50,7 +50,7 @@
   private Thread myriadExecutorThread;
   // Storing container id strings as it is difficult to get access to
   // NodeManager's NMContext object from an auxiliary service.
-  private Set<String> containerIds = new HashSet<>();
+  private Set<String> containerIds = new HashSet<String>();
 
```

Comment:
From what I've seen most people are preferring to use <> on the RHS as it's pretty redundant.  Not sure I think we should change these.